site/src/_pages/docs/1_developing-blocks.mdx

```diff
@@ -217,7 +217,7 @@ If you want to write blocks **using other technologies or frameworks**, you have
 
 1.  use a `custom-element` template and use different approaches when constructing the element-i.e. use a custom element as a wrapper for anything else you like.
 1.  use an `html` template and import and use your additional libraries inside the `<script>` tag.
-1.  use a React component as a wrapper around something else. As an example, see [this blog post](https://hash.dev/blog/build-blocks-in-any-language) on writing a block in F#, wrapped as a React component.
+1.  use languages which can be transpiled to JavaScript. As an example, see [this blog post](https://hash.dev/blog/build-blocks-in-any-language) on writing a block using F#. This block uses React, you can use transpiled JavaScript in blocks defined as HTML files or custom elements too.
```

Comment:
Non blocking, just think this would read better like this:
```suggestion
1.  use a language which can be transpiled to JavaScript. As an example, see [this blog post](https://hash.dev/blog/build-blocks-in-any-language) on writing a block using F#. This block uses React, you can use transpiled JavaScript in blocks defined as HTML files or custom elements too.
```